NAMESPACE

```diff
@@ -153,6 +157,7 @@ importFrom(grid,unit)
 importFrom(htmltools,tagList)
 importFrom(htmltools,tags)
 importFrom(stats,na.omit)
+importFrom(tidytlg,roundSAS)
```

Comment:
isnt it possible to copy? it is probably not worth the import

Reply:
roundSAS also relies on stringr, I will need some further investigation how to resolve this one